README.md

```diff
@@ -1,5 +1,80 @@
-# Aapt2 package build workflow
+This repo is based on the work of lzhiyong at https://github.com/lzhiyong/android-sdk-tools and the modification of aapt by IBotPeaches for apktool: https://github.com/aosp-mirror/platform_frameworks_base/compare/main...iBotPeaches:platform_frameworks_base:apktool_14.0.0
 
-[![build workflow](https://github.com/revanced/aapt2/actions/workflows/build.yml/badge.svg)](https://github.com/revanced/aapt2/actions/workflows/build.yml)
+Building platform-tools and build-tools for Android, such as `aapt aapt2 aidl zipalign adb fastboot` ... etc.
 
-This repository contains a workflow to build the aapt2 binaries for Android.
+If you need other tools, please refer to existing tools to add CMake file.
+
+Currently, only aarch64 has been tested, and it is not guaranteed to work for other architectures.
+
+In order to save storage and bandwidth, I have removed all submodules and no longer provide source code compressed packages.
+
+I believe that as long as you have the C/C++ and CMake foundation, it can be easily solved by referring to the patch files.
+
+ **** 
+ 
+### How to build
+First, we need to use `get_source.py` to clone the source code of all repositories.
+
+Please note that patch files only apply to specific branches or tags, so we may need to patch manually.
+
+```bash
+# default is master branch
+python get_source.py
+
+# specific branches or tags
+python get_source.py --tags platform-tools-35.0.2
+
+# or tags xxx
+# see available tags https://android.googlesource.com/platform/frameworks/base/+refs
+python get_source.py --tags xxx
+
+```
+
+ **** 
+
+Second, we need to build `protobuf` for the host OS which will be used to generate the required sources.
```

Comment:
Isn't there a package for ubuntu we can install/use without having to build it ourselves?